### ISSUE TITLE
Add Saint Lucia holidays

### ISSUE THREAD

**Issue by arkid15r** [vacanza/holidays#1262]
Links:
  - https://en.wikipedia.org/wiki/Public_holidays_in_Saint_Lucia
  - https://www.timeanddate.com/holidays/saint-lucia/
  - https://archive.stlucia.gov.lc/saint_lucia/public_holidays.htm

**Comment by Prateekshit73** [comment]
As suggested by @PPsyrius , I would like to work on this issue.

**Comment by PPsyrius** [comment]
@Prateekshit73 thanks for taking up this task ♥️

**Comment by Prateekshit73** [comment]
Other than the website suggested by @arkid15r, I found only one government website referencing LC holidays, namely : [National Holidays](https://archive.stlucia.gov.lc/stluciasilver/national_holidays.htm)

**Comment by Prateekshit73** [comment]
> As always I would recommend relying on 1st-party references first i.e. St. Lucia gov website, then use 3rd party ones for cross-checking.
@PPsyrius, should I work based on the three links provided by @arkid15r?

**Comment by PPsyrius** [comment]
@Prateekshit73 Yes, the code itself should probably be based on the law itself i.e. https://attorneygeneralchambers.com/laws-of-saint-lucia/bank-holidays-act/schedule

**Comment by Prateekshit73** [comment]
@PPsyrius, I need help. I came across the holiday `Thanksgiving Day`. Now, two different websites state different dates for the same holiday, as stated on [Timeanddate](https://www.timeanddate.com/holidays/saint-lucia/thanksgiving-day) and [Calendarific](https://calendarific.com/holiday/us/thanksgiving-day).

**Comment by PPsyrius** [comment]
> [@PPsyrius](https://github.com/PPsyrius), I need help. I came across the holiday `Thanksgiving Day`. Now, two different websites state different dates for the same holiday, as stated on [Timeanddate](https://www.timeanddate.com/holidays/saint-lucia/thanksgiving-day) and [Calendarific](https://calendarific.com/holiday/us/thanksgiving-day).

@Prateekshit73 I double-checked just to be sure, and it's [`1st Monday of October`](https://diversityatlas.io/events/thanksgiving-day-saint-lucia/) for St. Lucia. The Times n Date one is correct here - the Calendarific source you cited here is the US one.

```python
self._add_holiday_1st_mon_of_oct("Thanksgiving Day")
```
